geth/db/peers.go

```diff
@@ -0,0 +1,65 @@
+package db
```

Comment:
I would move this and peerspool.go to `geth/peers/` package.

Reply:
But that way you are spreading a single feature (a peer pool) around multiple packages.
If we'd need some common DB utilities, that are shared across multiple features, I agree we would need some `common/db/utils.go`. But right now, it is only one feature that uses it.
It would be nice to have it self-contained inside one package.

Reply:
ok, i can move to a single package